scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -15,8 +15,8 @@ import org.scalafmt.util.ValidationOps
 /** Configuration options for scalafmt.
   *
   * @param version
-  *   The version of scalafmt to use for this project. Currently not used, the
-  *   plan is to use this field for the IntelliJ+sbt integrations.
+  *   The version of scalafmt to use for this project. Must match the current
+  *   binary version.
```

Comment:
What do you mean by binary version? Scala binary version?

Reply:
how should i rephrase it? i meant the version of scalafmt that is running (aka, no dynamic resolution at this stage).

Reply:
just maybe `Must match the current version`?